src/server.cpp

```diff
@@ -2309,7 +2309,7 @@ void Server::sendMetadataChanged(const std::list<v3s16> &meta_updates, float far
 
 		// Send the meta changes
 		std::ostringstream os(std::ios::binary);
-		meta_updates_list.serialize(os, client->net_proto_version, false, true);
+		meta_updates_list.serialize(os, client->net_proto_version, false, true, true);
```

Comment:
Serialization and protocol version are very different things, it happens to work here in practice but it's technically broken
```suggestion
		meta_updates_list.serialize(os, client->client->serialization_version, false, true, true);
```